website/src/tech-guide.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: layouts/page.njk
+title: CiviForm | Technical Readiness Guide
+---
+# CiviForm Technical Readiness Guide
+
+## Overview
+
+CiviForm is a powerful open-source tool for governments that helps simplify the administration of benefit programs and makes it easier for residents to access those benefits. It lets residents and community organizations find and apply to multiple services in one place.
+
+This doc is for a technical audience. Whoever will deploy CiviForm can use this  guide to learn how to deploy the tool on a cloud provider. It includes an overview and a detailed guide of all steps needed at each stage.
```

Comment:
the browser will ignore it but there's extra whitespace between `this` and `guide`.

Reply:
Fixed in the original. It will fix the next time I do a PR for this doc. Thanks!